File: WindowsFormsApplication1/WindowsFormsApplication1/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace WindowsFormsApplication1
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void txt_fullname_Enter(object sender, EventArgs e)
        {
            txt_fullname.Text = txt_fname.Text + " " + txt_mname.Text + " " + txt_lname.Text;

        }

        private void txt_fullname_TextChanged(object sender, EventArgs e)
        {

        }
    }
}

File: WindowsApplication1/WindowsApplication1/Form1.vb
Public Class Form1

    Private Sub TextBox2_TextChanged(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles txt_mname.TextChanged

    End Sub

    Private Sub txt_fullname_Enter(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles txt_fullname.Enter
        txt_fullname.Text = txt_fname.Text + " " + txt_mname.Text + " " + txt_lname.Text


    End Sub
End Class
